modules/classes.js

```diff
@@ -0,0 +1,48 @@
+/* eslint-disable */
```

Comment:
- [ ] Please do not disable eslint as it analyzes your source code to flag programming errors, bugs, stylistic errors, and suspicious constructs, remove this eslint disable comment and try to fix those issues, for some issues such as using multiple classes etc.. you can disable only that particular rule, but to disable the whole thing loses the point of following best practices.